tests/unit_tests/steps/test_link.py

```diff
@@ -34,16 +34,20 @@ def test_run(self, tool_box):
             linker = Linker("mock_link", "mock_link.exe", "mock-vendor")
             # Mark the linker as available to it can be added to the tool box
             linker._is_available = True
+
+            # Add a custom library to the linker
+            linker.add_lib_flags('mylib', ['-L/my/lib', '-mylib'])
```

Comment:
If we are trying to be accurate to expected usage should this be

```suggestion
            linker.add_lib_flags('mylib', ['-L/my/lib', '-lmylib'])
```

Reply:
Oh yes, indeed.